tests/testcases/Group8-Configuration/8-01-Authentication-edit-authentication.md

```diff
@@ -14,11 +14,16 @@ User guide
 
 # Test Steps:
 
-TBD
+1. Login UI as admin user.  
+2. In configuration page,change authentication mode from DB to LDAP or from LDAP to DB.  

```

Comment:
This needs a precondition that there is only admin user in the system, or the authentication can not be changed.